### PR TITLE
Data Modeling

### PR DESCRIPTION
**Pull request by Vaibhavpratapsingh22**
Chapter- 2 created 👍

### CODE REVIEW

**Comment by udeysingh95** [comment]
@siddharth2848 could you review the content here once - since you have a good hold on DB topic?

**Comment by Vaibhavpratapsingh22** [comment]
@udeysingh95 1,2,3 are for main points and a,b,c are for sub points.

File: 5-dbms/2-data-modeling.md [hunk 1]
```diff
@@ -0,0 +1,237 @@
...
+
+   <img src="https://media.geeksforgeeks.org/wp-content/uploads/Database-Management-System-ER-Model-4.png" height="" width="">
+
+   c. **Multi-valued Attribute**- An attribute can have more than one value. These attributes are known as a multi-valued attribute. The double oval is used to represent multi-valued attribute.
+
+   <img src="https://i.stack.imgur.com/B2lxV.png" height="" width="">
+
+   d. **Derived Attribute**- An attribute that can be derived from other attribute is known as a derived attribute. It can be represented by a dashed ellipse.
+
+   <img src="https://www.gatevidyalay.com/wp-content/uploads/2018/06/Derived-Attributes-Example.png" height="" width="">
```
[the remark this answers is not in the document]
Comment (Vaibhavpratapsingh22) [reply 5-dbms/2-data-modeling.md:198]: YES, I can make them in draw.io.

[the remark this answers is not in the document]
Comment (udeysingh95) [reply 5-dbms/2-data-modeling.md:198]: That would take too much effort. Find some other way. This whole course is pretty big - working on draw.io as well - is not sustainable. Stick to google images - but maybe joins two or three together sometime and put them here as one only. What do you say?

[the remark this answers is not in the document]
Comment (Vaibhavpratapsingh22) [reply 5-dbms/2-data-modeling.md:198]: @udeysingh95  Let's give it a try and see how much time does it takes.

**Pull request merged**